sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -343,6 +317,75 @@ def GetHeaders(  # pylint: disable=too-many-statements,too-many-branches
 
     return headers
 
+def _is_session_token_request(
+        cosmos_client_connection: Union["CosmosClientConnection", "AsyncClientConnection"],
+        headers: dict,
+        request_object: "RequestObject") -> bool:
+    consistency_level = headers.get(http_constants.HttpHeaders.ConsistencyLevel)
+    # Figure out if consistency level for this request is session
+    is_session_consistency = consistency_level == documents.ConsistencyLevel.Session
+
+    # Verify that it is not a metadata request, and that it is either a read request, batch request, or an account
+    # configured to use multiple write regions
+    return (is_session_consistency is True and not IsMasterResource(request_object.resource_type)
```

Comment:
How do we validate here that the account level consistency is at least `Session`? Request-level consistency can only apply when it is a Consistency downgrade.